build-support/bin/generate_github_workflows.py

```diff
@@ -329,6 +350,68 @@ def test_workflow_jobs(primary_python_version: str) -> Jobs:
             ],
         },
     }
+    if not cron:
+        build_wheels_step = {
+            "name": "Build wheels and fs_util",
+            "run": dedent(
+                # We use MODE=debug on PR builds to speed things up, given that those are only
+                # smoke tests of our release process.
+                """\
+                [[ "${GITHUB_EVENT_NAME}" == "pull_request" ]] && export MODE=debug
+                ./build-support/bin/release.sh -n
+                USE_PY38=true ./build-support/bin/release.sh -n
+                ./build-support/bin/release.sh -f
```

Comment:
This is probably unnecessary on PR builds, and we could save lots of resources by using `./cargo check`. Can look into as a followup.